apis/nucleus/src/components/Sheet.jsx

```diff
@@ -17,10 +17,13 @@ const SheetElement = {
   className: 'njs-sheet',
 };
 
-function getCellRenderer(cell, halo, initialSnOptions, initialSnPlugins, initialError, onMount) {
+function getCellRenderer(cell, halo, initialSnOptions, initialSnPlugins, initialError, onMount, navigation) {
   const { x, y, width, height } = cell.bounds;
   return (
-    <div style={{ left: `${x}%`, top: `${y}%`, width: `${width}%`, height: `${height}%`, position: 'absolute' }}>
+    <div
+      style={{ left: `${x}%`, top: `${y}%`, width: `${width}%`, height: `${height}%`, position: 'absolute' }}
+      key={cell.model.id}
```

Comment:
Whats with the key here?

Reply:
This is a list of cells, without a key it will use index and this causes a problem when we switch from a sheet to another sheet. Cells in a new list are not rendered, instead cells in old list are still used.